tests/dashboards/scenarios/case_yaml_ref_dashboard.yaml

```
@@ -6,15 +6,15 @@ dashboards:
   name: Pie Chart Only on Aerospike Namespace with yaml ref
   panels:
   - title: "Pie Chart of Aerospike Namespace" 
-    type: charts
+    type: lens
     grid: { x: 0, y: 0, w: 24, h: 15 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)